src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4351,7 +4352,7 @@ static void WindowRideColourMousedown(rct_window* w, WidgetIndex widgetIndex, rc
     switch (widgetIndex)
     {
         case WIDX_TRACK_COLOUR_SCHEME_DROPDOWN:
-            for (i = 0; i < OpenRCT2::Limits::NumColourSchemes; i++)
+            for (i = 0; i < OpenRCT2::Limits::NumColourSchemes + 1; i++)
```

Comment:
I think this should not be changed, since you already incremented OpenRCT2::Limits::NumColourSchemes in Limits.h. (I didn't complie or run this so I'm kind of guessing)